onboarding-manager/internal/tinkerbell/status.go

```diff
@@ -23,6 +23,8 @@ var WorkflowStepToStatusDetail = map[string]string{
 	ActionSystemConfiguration:      "Applying system configuration",
 	ActionKernelupgrade:            "Upgrading kernel",
 	ActionCloudInitInstall:         "Installing cloud-init",
+	ActionCustomConfigInstall:      "Copying custom cloud-init configs",
+	ActionCustomConfigSplit:        "Installing custom cloud-init configs",
```

Comment:
can we do with a single action? Adding an action has a scalability cost

Reply:
Attempted a few other approaches where the number of actions would be equal to the number of custom configs but that would have been even worse from this perspective. Another approach(more scalable) is to have a custom action to copy multiple files in a single action. Scoping that for a future enhancement.